migration/v0.47/apply/80-apply.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+# Add selector filters if covered by other snippets.
+# Example: "app!=something"
+declare -a skipped
+skipped=(
+)
+declare -a skipped_sc
+skipped_sc=(
+)
+declare -a skipped_wc
+skipped_wc=(
+  "app!=prometheus"
+  "app!=app=dev-rbac"
```

Comment:
This looks like a miss. Is this suggestion correct?
```suggestion
  "app!=dev-rbac"
```

Reply:
Done with the latest commit.